### PR TITLE
Add black formatter

### PR DESCRIPTION
**Pull request by TheTripleV**

### CODE REVIEW

File: tests/test_options.py [hunk 1]
```diff
@@ -95,4 +100,3 @@ def test_skip_comments(og_meta_tags):
 @pytest.mark.sphinx("html", testroot="custom-tags")
 def test_custom_tags(og_meta_tags):
     assert get_tag_content(og_meta_tags, "ignore_canonical") == "true"
-
```
Comment (Daltz333) [inline tests/test_options.py:98]: EOL at EOF

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -10,8 +10,21 @@ on:
...
+      with:
+        python-version: 3.8
+    - name: Black
+      run: |
+        pip install black
+        black --check --diff .
+        
   test-extension:
     runs-on: ubuntu-latest
+    needs: [check]
```
Comment (Daltz333) [inline .github/workflows/workflow.yml:27]: We should still run tests even if the check fails

Comment (TheTripleV) [reply .github/workflows/workflow.yml]: done

**Comment by Daltz333** [comment]
I notice on some repos there is a black GitHub badge, could we add that?

**Comment by TheTripleV** [comment]
> I notice on some repos there is a black GitHub badge, could we add that?

done

**Pull request merged**